packages/utilities/src/FeatureFlags.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+/**
+ * FeatureFlags allows us to ship features off by default, unless the consuming app turns on that feature. This is
+ * intended to let a partner test a given feature first before releasing it to the rest of FluentUI consumers.
+ * The flags are provided via React Context.
+ */
+
+type FeatureFlags = {
+  ModalAriaHidden?: boolean;
+  TestFeature?: boolean; // used for testing only, replace with a real feature in .test when a second feature is added
+};
+
+const _featureFlags: FeatureFlags = {};
+
+export const FeatureFlagsContext = React.createContext<FeatureFlags>(_featureFlags);
```

Comment:
This can be `React.createContext<Record<string, boolean>>`

In OUFR, you can export this:

```tsx
export interface FeatureFlagsProvider {
  /**
   * Comment here explaining feature, who added it, what scenarios need it.
   */
  modalAriaHidden?: boolean;
}

export const FeatureFlagsProvider = (props: React.PropsWithChildren<FeatureFlagsProviderProps>) => {
  const { children, ...rest } = props;
  return (
    <FeatureFlagsContext.Provider value={ rest }>{ children }</FeatureFlagsContext.Provider>
  );
};

This means customers simply use the provider:
```tsx
<FeatureFlagsProvider modalAriaHidden>
  <App/>
</FeatureFlagsProvider>
```